genienlp/train.py

```diff
@@ -165,7 +165,7 @@ def train_step(model, batch, iteration, opt, devices, lr_scheduler=None, grad_cl
         opt.zero_grad()
     loss, predictions = model(batch, iteration, pretraining=pretraining)
     if torch.isnan(loss).any():
-        raise RuntimeError('Got NaN loss')
+        raise RuntimeError('Got NaN loss %s', str(loss))
```

Comment:
Wouldn't `str(loss)` be `nan`? Why do you want to print it?

Reply:
I don't quite remember how this was a problem, but the difference is that on multiple GPUs, some elements of loss might be NaN, not all of them, and we are using `.any()`.